test/mavsdk_tests/autopilot_tester.h

```diff
@@ -283,7 +299,11 @@ class AutopilotTester
 	}
 
 
+<<<<<<< HEAD
 	mavsdk::Mavsdk _mavsdk{Mavsdk::Configuration{ComponentType::GroundStation}};
+=======
+	mavsdk::Mavsdk _mavsdk{};
+>>>>>>> 5a03cfb986e (mavsdk tests: use tester sleep function)
```

Comment:
This needs updating the v3 ctor. I think the top one is right.

See: https://mavsdk.mavlink.io/main/en/cpp/api_changes.html#mavsdk-configuration